tests/acceptance/stepDefinitions/generalContext.js

```diff
@@ -58,6 +58,13 @@ Then('the error message {string} should be displayed on the webUI dialog prompt'
     .expect.element('@ocDialogPromptAlert').text.to.equal(message)
 })
 
+Then('the user should see the following error message on the login card dialog', function (message) {
+  return client
+    .page.publicLinkPasswordPage()
+    .waitForElementVisible('@loginCardDialogBox')
+    .expect.element('@loginCardDialogBox').text.to.equal(message)
```

Comment:
It's better not use the selector names directly from the context files, rather create a method in the page object that returns the error message, but since we have other cases like this we can do that in different PR.